content/lectures/final_project.md

```diff
@@ -0,0 +1,39 @@
+Title: Final Project Instructions
+Summary:  Final Project
+Date: 10/31/2017
+Tags: final, project
+Category: lectures
+Author: Kerry Key and Ryan Abernathey
+
+
+
+** Final Project **
+
+
+The final project is intended to be an independent project that uses the skills you have learned in this class, as well as things you have learned on your own, to create a complicated and interesting data analysis program that accomplishes one or more research-relevant tasks. Unlike the homework assignments where we supplied the data files and told you what to do, now it is your turn to identify a  data set relevant to your own research interests and to create a code that will analyze that data and present  results in graphical form.
+
+***Grading***
+
+The final project will be worth 40% of your class grade so you should expect to put a good amount of time and effort into it.  
+
+ In order to get the maximum credit for the final project you will need to:
+
+1. Turn in a project proposal by **Thursday, November 9**. The proposal description only needs to be a paragraph long. It should describe in words what your final project will do and what programming languages you intend to use. We will review your proposal ASAP and let you know if it is acceptable.  Place your proposal description in a file call `proposal.txt` and turn it via github in a folder call `FinalProject`.
+
+2. Your project will need to **use at least two different main topics covered in this class and it must contain a visualization component**. For example, your project could extract data from GeoMapApp, analyze it using Pandas and then create a new image resulting from the data analysis. Another example is to analyze 3D array data with Xarray and then create a map plot or visualization using Python or GMT.
+
+3. ** Document and comment your code**.   Since this is your project and not an ordinary exercise, we may not know what you are trying to do. Documentation will allow us to better understand your intentions. Add comments to your codes to inform of us your intentions, if it is not easily discerned from the specific code commands.  If your code analyzes data that were downloaded from the internet or given to you by someone, state where you obtained the data (name of organization or person) and the URL where the data can be downloaded if from the internet.  
+
```

Comment:
"If you use python, your project should take the form of a heavily annotated notebook."